pandas/tests/test_strings.py

```diff
@@ -3670,3 +3670,11 @@ def test_str_get_stringarray_multiple_nans():
     result = s.str.get(2)
     expected = Series(pd.array([pd.NA, pd.NA, pd.NA, "c"]))
     tm.assert_series_equal(result, expected)
+
+
+def test_str_accessor_in_apply_func():
+    # https://github.com/pandas-dev/pandas/issues/38979
+    df = pd.DataFrame(zip("abc", "def"))
+    expected = df.apply(lambda f: "/".join(f), axis=1).str.upper()
```

Comment:
might be worth writing this out explicitly?

Reply:
good call.